src/Core/FormatFactorySettings.h

```diff
@@ -1242,6 +1242,9 @@ Set the quoting rule for identifiers in SHOW CREATE query
 )", 0) \
     DECLARE(IdentifierQuotingStyle, show_create_query_identifier_quoting_style, IdentifierQuotingStyle::Backticks, R"(
 Set the quoting style for identifiers in SHOW CREATE query
+)", 0) \
+    DECLARE(Bool, spark_text_output_format, false, R"(
```

Comment:
Maybe we can introduce it in a bit more generic way, like `composed_data_type_output_format_mode = 'default'/'spark'` so we can support more modes in the future and not making `spark` special case

Reply:
Good point. I will try later.